src/core/qgsprojectstorageregistry.cpp

```diff
@@ -52,5 +52,5 @@ void QgsProjectStorageRegistry::registerProjectStorage( QgsProjectStorage *stora
 
 void QgsProjectStorageRegistry::unregisterProjectStorage( QgsProjectStorage *storage )
 {
-  delete mBackends.take( storage->type() );
+  mBackends.remove( storage->type() );
```

Comment:
Does remove delete though? I don't think it does.

Reply:
oops... sorry for that!